### PR TITLE
LPS-164092 Pass notificationTemplateType to EditNotificationTemplate component

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**

### CODE REVIEW

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

File: modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/display/context/ViewNotificationTemplatesDisplayContext.java [hunk 1]
```diff
@@ -76,17 +79,16 @@ public CreationMenu getCreationMenu() {
...
-					"backURL", _notificationRequestHelper.getCurrentURL());
-				dropdownItem.setLabel(
-					LanguageUtil.get(
-						_notificationRequestHelper.getRequest(),
-						"add-notification-template"));
-			});
+		_addDropdownItem(
+			creationMenu, "email", NotificationConstants.TYPE_EMAIL);
+
+		if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-162245"))) {
```
Comment (alinedoleron) [inline modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/display/context/ViewNotificationTemplatesDisplayContext.java:85]: Hi @MarinhoFeliphe, is this feature flag is correct? I think it should be the LPS of the current story: [LPS-162133](https://issues.liferay.com/browse/LPS-162133)

Comment (MarinhoFeliphe) [reply modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/display/context/ViewNotificationTemplatesDisplayContext.java]: @alinedoleron

my bad!, I changed to the right LPS

Thank you!

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Review by rodrigopaulino — APPROVED** [review]
LGTM! 🎉

**Comment by rodrigopaulino** [comment]
Hey, @MarinhoFeliphe. This just a need a good old rebase. Feel free to call the forward after. Thx!

**Comment by MarinhoFeliphe** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by MarinhoFeliphe** [comment]
ci:stop

**Comment by MarinhoFeliphe** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/123881

**Pull request closed without merging**